DigitalLearningSolutions.Web/Views/TrackingSystem/Delegates/ViewDelegate/ConfirmRemoveFromCourse.cshtml

```diff
@@ -0,0 +1,47 @@
+﻿@inject IConfiguration Configuration
+@using DigitalLearningSolutions.Web.ViewModels.TrackingSystem.Delegates.ViewDelegate
+@using Microsoft.Extensions.Configuration
+@model RemoveFromCourseViewModel
+
+@{
+  var errorHasOccurred = !ViewData.ModelState.IsValid;
+  ViewData["Title"] = $"Remove delegate enrolment from {Model.CourseName}";
+  ViewData["Application"] = "Tracking System";
+  ViewData["HeaderPath"] = $"{Configuration["AppRootPath"]}/TrackingSystem/Centre/Dashboard";
+  ViewData["HeaderPathName"] = "Tracking System";
+}
+
+@section NavMenuItems {
+  <partial name="~/Views/TrackingSystem/Shared/_NavMenuItems.cshtml" />
+}

```

Comment:
We should be using the new ApplicationType enumeration for these ViewData values now, and using the AutoNavMenuItems partial.

```
ViewData["Application"] = ApplicationType.TrackingSystem.HeaderExtension;
ViewData["HeaderPath"] = ApplicationType.TrackingSystem.HeaderPath;
ViewData["HeaderPathName"] = ApplicationType.TrackingSystem.HeaderPathName;
```